basic/s3_log_destination/s3_log_destination.tf

```diff
@@ -0,0 +1,65 @@
+# The AWS dependencies required to declare a Kinesis Firehose, such as the
+# IAM role the Firehose will assume and the backup S3 bucket.
+# This is not required, as it is just an abstraction of the dependencies.
+# You may declare these resources manually if you wish.
+module "kinesis_firehose_connector" {
+  source  = "symopsio/kinesis-firehose-connector/aws"
+  version = ">= 3.0.0"
+
+  environment = "main"
+}
+
```

Comment:
Needs the same `firehose.tf` as the datadog example

https://docs.symops.com/v5.0/docs/logs-to-s3